vehicles-utils/upload-insurances/upload-insurances.py

```
@@ -47,8 +149,18 @@ def process_excel_files():
+                        vehicle_id = get_vehicle_id(row["Matrícula"], vehicles)
+
+                        mapped_data = try_to_map(row, suppliers, insurance_types)
+
+                        update_vehicle(vehicle_id, mapped_data)
+
                         processed_rows.append(mapped_data)
+
+                        logging.info(
+                            f"Esperando {MAX_SECONDS_TO_SLEEP} segundos antes de seguir con el siguiente registro."
+                        )
+                        time.sleep(MAX_SECONDS_TO_SLEEP)
```

Ruff (0.8.2)

Undefined name `time`

(F821)